tools.md

```diff
@@ -3,20 +3,19 @@ layout: page
 title: General Purpose Tools
 ---
 
-### estimagic
-estimagic is a Python package to fit large-scale empirical models to data and make
-inferences about the estimated model parameters. It is especially suited to solve
-difficult optimization problems. It provides several advantages over similar packages,
-including a unified interface that supports a large number of local and global
-optimization algorithms and the possibility of monitoring the optimization procedure via
-a beautiful interactive dashboard. It provides tools for nonlinear optimization,
-numerical differentiation, and statistical inference.
-[(Documentation)](https://estimagic.readthedocs.io){:target="_blank"}
+### optimagic
+optimagic is a Python package for numerical optimization. It is a unified interface to
+optimizers from SciPy, NlOpt and many other Python packages. optimagic’s `minimize`
+function works just like SciPy’s, but you get more optimizers for free. On top you get
+powerful diagnostic tools, parallel numerical derivatives and more. optimagic was
```

Comment:
Something like: "We started developing optimagic under the name estimagic out of a need to robustify estimation of nonlinear economic models, including statistical inference. That functionality is now a subpackage of optimagic, which is still called estimagic. Optimagic now has its own organization on Github: [optimatic-dev](link)."

?

Reply:
I like it. Do you imagine this text as a replacement for the current text or as a preamble?

Reply:
Sorry, formatting was not so nice. I would actually have left the original, replacing "optimagic was formerly ... and statistical inference." with what I wrote.